templates/zerver/app/navbar.html

```diff
@@ -104,6 +104,12 @@
                                 </a>
                             </li>
                             {% endif %}
+                            <li role="presentation">
+                                <a href="https://zulip.readthedocs.io/en/latest/overview/changelog.html"  target="_blank" rel="noopener noreff" role="menuitem" class="menuitem-version">
+                                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="49.99 49.99 673.14 673.14"><path d="M688.52 150.67c0 33.91-15.23 64.04-38.44 82.31L424.79 434.17c-4.18 3.59-9.62-2.19-6.61-7.03l82.64-165.46c2.31-4.63-.69-10.33-5.44-10.33H174.86c-49.64 0-90.26-45.31-90.26-100.68 0-55.37 40.62-100.68 90.26-100.68h423.39c49.65 0 90.27 45.31 90.27 100.68zM174.86 723.13h423.39c49.64 0 90.26-45.31 90.26-100.68 0-55.37-40.62-100.68-90.26-100.68H277.73c-4.75 0-7.76-5.7-5.44-10.33l82.64-165.46c3.01-4.83-2.42-10.62-6.61-7.03L123.04 540.14c-23.21 18.27-38.44 48.4-38.44 82.31 0 55.37 40.62 100.68 90.26 100.68z" /></svg>
+                                    {{ _('Zulip ' + (zulip_version[:-4] if zulip_version.endswith('+git') else zulip_version) ) }}
```

Comment:
I think we should generate the URL and display text for this in Python code -- we try hard to avoid complex logic in templates since it can obscure the code, and I think we may want multiple conditionals over time, and we definitely want tests for the function mapping `zulip_version` to the string + URL.

(E.g. we might want the URL to link to the right section of the changelog).

Also, there's no reason to tag "Zulip" for translation :)

Reply:
And tagging a computed string for translation won’t work even if there were a reason.